src/components/container-with-background.tsx

```diff
@@ -3,34 +3,46 @@ import { Box, Container } from 'theme-ui';
 export interface ContainerWithBackgroundProps {
   color?: 'green' | 'beige';
   element?: 'element1' | 'element2' | 'element3' | 'element4' |'element5' | 'element6';
+  marginbottom?: '-170px';
   positionx?: 'right' | 'left';
-  positiony?: '0' | '10' | '20' | '25' | '30' | '32' | '40' |'50' | '60' |'70' | '80' |'90';
-  paddingtop?: '40px' | '190px' | '210px';
-  paddingbottom?: '210px';
+  positiony?: '0' | '10' | '20' | '25' | '30' | '32' | '40' | '44' | '46' | '50' | '60' |'70' | '80' |'90';
+  paddingtop?: '40px' | '50px' | '70px';
+  paddingbottom?: '90px';
```

Comment:
Maybe just pass the `sx` prop? Or use `React.forwardRef`?

Reply:
Yeah, that sounds a lot easier - but I still wanted to only let some values pass as viable.